chirp/common.py

```diff
@@ -140,3 +143,18 @@ def wait() -> None:
         else:
             os.system('read -s -n 1 -p "Press any key to continue..."')  # nosec
             print()
+
+
+def iocs_discovered() -> bool:
+    """Determine whether iocs were discovered."""
+    report_files = glob.glob("{}/*".format(OUTPUT_DIR))
+    for report_file in report_files:
+        with open(report_file, "r") as f:
+            data = json.load(f)
+            if len(data) > 0:
+                logging.log(
+                    70, "Discovered IoC's, please see output reports for more details."
```

Comment:
As we're using the `COMPLETED` code (70) for these messages, we probably want to change our default log level when non-interactive mode is invoked to only display 70+ logs, leaving the `-v` switch for verbose-ness if a user wants to run in non-interactive mode but wants full output.